misc/docker-ci/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+# To build Docker images for CI.
+# https://hub.docker.com/r/h2oserver/h2o-ci
+
+set -xe
+
+IMAGE_NAME=h2oserver/h2o-ci
+REVS="ubuntu1604 ubuntu2004"
+
+PROJECT_ROOT_DIR=$(realpath "$(dirname $0)/../..")
+cd $PROJECT_ROOT_DIR
```

Comment:
IIUC, if `docker build` is invoked in the project root directory, all the contents of the root directory will become part of the docker image. That is not only a waste of disk space but also poses a security risk, as people might have some secret data under the project root directory.

We should run `docker build` from an empty directory.

Reply:
Whoa, I misunderstood it! It sounds makes sense, and`.dockerignore` doesn't mitigate the security risk because it is a deny-list, not an allow-list, right?

Okay.  How about this? 294fd502aea4841a9f0c8a8412ee93f4dc298be5 (that includes the fix for the other comment).